pom.xml

```diff
@@ -44,10 +45,11 @@
     <module>fuzz</module>
     <module>jmh</module>
     <module>log</module>
+    <module>machines-tests</module>
```

Comment:
Nit: `machine-tests` sounds nicer. The double plural is a bit off.

Reply:
I couldn't decide for an appropriate name for this module testing the interoperability of the implementations.
Let's go with `machine-tests` for now :+1: